projects/framework-nuke/release_notes.md

```diff
@@ -1,5 +1,10 @@
 # ftrack Framework Nuke integration release Notes
 
+## Upcoming
+
+* [new] PySide6 support.
+* [changed] Removed Qt.py dependency.
```

Comment:
We never had qt.py as dependnecy on the framework projects, should we add PySide6 and 2 as dependencies? we where alwys counting that framework-qt already had that dependency, but not sure if it is a good approach, is simpler to maintain, but it might be a bit confusing.